GCTC-NTGC/gc-digital-talent
# Remove search field for Pools table

Resolves #4280.

Modifies the Table component to allow removing the search field and removed it on the Pools table.

To test:
- Navigate to the Pools table and confirm it no longer has a search field
- View some other tables and ensure they still have the search field if they did before.
